src/interfaces/SetParameter.ts

```diff
@@ -1,4 +1,4 @@
-import Parameter from './Parameter';
+import { Parameter } from './Parameter';
```

Comment:
I changed `Parameter.ts` to do an `export default`. Does this still work, or do you have to use `import Parameter from './Parameter';`?